modules/template/services/TemplateInstanceRendererService.php

```diff
@@ -34,8 +35,12 @@ public function __construct(CustomPage $customPage)
         }
     }
 
-    public static function instance(CustomPage $customPage): self
+    public static function instance(CustomPage $customPage, ?string $mode = null): self
```

Comment:
@yurabakhtin Can we switch the string `$mode` here to something like `bool $enableEditMode`?